extension/manifest.json

```diff
@@ -75,7 +75,7 @@
       "js": ["shared.js","keysocket-grooveshark.js"]
     },
     {
-      "matches": ["https://www.amazon.com/gp/dmusic/mp3/player*"],
+      "matches": ["https://www.amazon.com/gp/dmusic/cloudplayer/web*"],
```

Comment:
This change does not seem to have resolved the issue for me. I still cannot control the Amazon player with this extension, also the extension's icon does not show up to the right of the address bar (next to he bookmark stat) like it does for other websites that do work.